src/Database/PostgreSQL/Simple/FromRow.hs

```diff
@@ -67,6 +67,20 @@ import           GHC.Generics
 -- in a single row of the query result.  Otherwise,  a 'ConversionFailed'
 -- exception will be thrown.
 --
+-- You can also derive 'FromRow' for your data type using GHC generics, like
+-- this:
```

Comment:
should one mention `Product` type? Should we add an instance with `TypeError` for `:+:`?

Reply:
@phadej: I think adding new instances is outside of the scope for this pull request (which is just to mention that it even exists)

However, I can update the documentation to mention that it only works for product types and does not support sum types